File: frontend/src/types/Note.ts
export type TNote = {
  id: string;
  createdBy: string;
  createdByName: string;
  noteText?: string;
  pathFile?: string;
  createdAt: string;
  updatedAt: string;
};

export type TAddNote = {
  customerId?: string;
  createdBy: string;
  createdByName: string;
  noteText?: string;
  pathFile?: string;
};

export type TUpdateNote = {
  id: string;
  noteText?: string;
  pathFile?: string;
};

File: backend/Models/Note.cs
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace backend.Models;

public class Note
{
    [Key]
    public Guid Id { get; set; }
    public string CreatedBy { get; set; } = string.Empty;
    public string CreatedByName { get; set; } = string.Empty;
    public string? NoteText { get; set; }
    public string? PathFile { get; set; }
    public Guid CustomerId { get; set; }
    [JsonIgnore]
    public Customer? Customer { get; set; }
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    public DateTime UpdatedAt { get; set; } = DateTime.UtcNow;
}